x-pack/plugins/index_lifecycle_management/public/application/sections/edit_policy/edit_policy.container.tsx

```diff
@@ -76,12 +76,22 @@ export const EditPolicy: React.FunctionComponent<Props & RouteComponentProps<Rou
     );
   }
 
+  let decodedPolicyName = policyName;
```

Comment:
Can we add a comment here that this code can be removed if we upgrade to react-router 6 and history 5?